dev.sparcles.SREngine.yml

```diff
@@ -0,0 +1,43 @@
+id: dev.sparcles.SREngine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srengine-run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home:create
+
+build-options:
+  build-args:
+    - --filesystem=home:create
+
+modules:
+   - name: srengine
+     buildsystem: cmake-ninja
+     config-opts:
+       - -DCMAKE_BUILD_TYPE=Release
+       - -DSR_ENGINE_FLATPAK_BUILD=ON
+       - -DSR_ENGINE_EXTENDED_NAME=OFF
+
+     build-commands:
```

Comment:
This needs to be in post-install

also why is cmake not installing the stuff?

Reply:
> This needs to be in post-install

Why? These are the CMake configuration flags. If I don't specify them here, the executable name will contain build info, e.g. `SREngine_v0.0.7-latest_linux.x86_64`. And without the first option, the entire build will fail.

Reply:
line 28 should be run by the cmake install target.

Reply:
>  These are the CMake configuration flags.

I'm talking about the build-commands.